frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -379,3 +379,49 @@ export const fetchMineDocuments = (mineGuid) => (dispatch) => {
       dispatch(hideLoading());
     });
 };
+
+// MineVerifcationStatus
+export const fetchMineVerifiedStatuses = (user_id) => (dispatch) => {
+  const params = user_id ? { user_id } : null;
+  dispatch(request(reducerTypes.GET_MINE_VERIFIED_STATUS));
+  dispatch(showLoading());
+  return axios
+    .get(`${ENVIRONMENT.apiUrl}${API.MINE_VERIFIED_STATUSES(params)}`, createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_MINE_VERIFIED_STATUS));
+      if (params) {
+        dispatch(mineActions.storeCurrentUserMineVerifiedStatuses(response.data));
+      }
+      dispatch(hideLoading());
+      return response;
+    })
+    .catch((err) => {
+      notification.error({
+        message: err.response ? err.response.data.error.message : String.ERROR,
+        duration: 10,
+      });
+      dispatch(error(reducerTypes.GET_MINE_VERIFIED_STATUS));
+      dispatch(hideLoading());
+    });
+};
+
+export const setMineVerifiedStatus = (mine_guid, healthy) => (dispatch) => {
```

Comment:
I think it would be beneficial for our action creators to follow a standardized input pattern going forward. I'd be in-favour of `actionCreatorName(urlParams: object, payload: object)` because that is the pattern used by axios already, and it is fully able to support 0, 1, or n urlParams and 0, 1, or n payload properties.

Thoughts? @mmumby ? I would see this as a fix-forward thing.

I think this works well with our use of `queryString` in the API endpoint constant functions, which have been gradually converted from accepting urlParam strings to param objects.

Reply:
If I am understanding this correctly, I believe I had an opposite suggestion in the comment above.

I'm all for standardization, @NWCalvank. 
I guess if we all agree on a standard put/post pattern, and using `queryString` in our API consts, than you can disregard the above comment @Jsyro

Reply:
Your suggestion is a reasonable refactor for the code as it currently is, but I think we should keep ourselves open to the idea that we may want to add more parameters to this endpoint in the future, which a params object more easily supports. Hope that makes sense :) 

And, as mentioned, it allows all of the endpoints to have the same argument pattern, which makes for less context-switching when moving to and from endpoints.

Reply:
Agreed! makes sense.

Reply:
I've made the action creator accept generic payload, I think that get's us to where we want to be, let me know if you want any more changes.